### DISCUSSION TITLE
2017/10/20/harveys-counterfeit-beacon-october-2017-update

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#6] [Announcements]
# 2017/10/20/harveys-counterfeit-beacon-october-2017-update

http://localhost:4000/2017/10/20/harveys-counterfeit-beacon-october-2017-update

**Comment by jengalas** [comment]
Posted by Dick Merrill
October 21, 2017 at 3:36 pm
_Imported from WordPress_

Nice job Harvey. Where is your airport? I would like to visit sometime soon.
